src/js/worker/core/player.js

```diff
@@ -668,15 +671,25 @@ function genRatings(
     season: number,
     scoutingRank: number,
     tid: number,
+	predeterminedHeight: number,
 ): PlayerRatings {
     let profileId;
-    if (profile === "Point") {
+    if (predeterminedHeight !== undefined) {
```

Comment:
I added the predeterminedHeight to the generate() function. I didn't go through to see if genRatings() is called from anywhere else.  Based on your comments here I assume not, but if so, then an undefined predeterminedHeight would mean it's a call from somewhere other than generate().  In that case genRatings() would simply work as before.